docs/how-to/add-delete-user-entity.md

```diff
@@ -32,15 +32,15 @@ An Authentication plugin requires an Authentication entity containing the proper
 
 By default, the 'User' entity is defined as the Authentication entity, but you can [configure it](#changing-the-default-authentication-entity). If the `User` entity is created automatically, it comes with a set of default fields necessary for the authentication plugins to work correctly:
 
-- `id` (required by all authentication plugins): An automatically created unique identifier of the entity. It is a required and unique field.
```

Comment:
@mulygottlieb - Can you please review the list and whether the "required" fields are indeed required for all the auth plugins (.NET and Node.js)?